remember rememer.py

```diff
@@ -0,0 +1,11 @@
+from PIL import Image
+im = Image.open('code.PNG','r')
+px = im.load()
+l=[]
```

Comment:
Variable names - https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0 - variable names, abbreviation reference is missing

'im' 'l' 'px' are not valid as I wrote in the previous ex